include/seastar/core/metrics_types.hh

```diff
@@ -54,6 +55,8 @@ struct histogram_bucket {
 struct histogram {
     uint64_t sample_count = 0;
     double sample_sum = 0;
+    std::optional<int32_t> schema;
```

Comment:
might want to add comment to explain how this is connected to protobuf's `Histogram` message definition? otherwise one can hardly figure out what `schema` implies.

Reply:
done